go.mod

```diff
@@ -99,3 +99,5 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20211105154855-cb9596dd5fba // release-4.10
 	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20211105081319-76d6155c1dab // release-4.10
 )
+
+replace sigs.k8s.io/scheduler-plugins => sigs.k8s.io/scheduler-plugins v0.22.6
```

Comment:
before using replace, I'd start bumping `scheduler-plugins` in the top-level `require` block

Reply:
I thought that using replace is a best practice convention, but sure i'll change it